docs/conf/portland/2025/news/announcing-cfp.rst

```diff
@@ -0,0 +1,70 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/portland-2024-speakers.jpg
+:og:image: _static/conf/images/headers/portland-2024-speakers.jpg
+
+.. post:: Oct 28, 2024
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Announcing the Call for Proposals and Sponsorship Prospectus
+============================================================
+
+We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year, even though the conference back at the start of May this time around.
+
+We're also launching our `Sponsorship Prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ for 2024 with all the information for sponsoring the conference.
+
+Read on for more details about how you can get involved this year.
+
+Call for Proposals
+------------------
+
+Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians.
+If there is something you'd really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We're looking for talk proposals to be delivered in-person, on-stage in Portland, on {{ date.short }}.
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__, and `submit your talk <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.
+
+The Call for Proposals will be open until **11:59 PM {{cfp.ends}} PST**, so harness that new year energy!
+
+(If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for our Atlantic conference held online in a timezone accessible to people all over the world).
+
+
+Sponsorship Updates
+-------------------
+
+Our `Sponsorship Prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ is updated for {{ year }},
+with various improvements based on sponsor feedback.
+
+We're focussing on in-person interaction at the conference, taking full advantage of {{ about.venue }} with plenty of space for sponsors to interact with attendees.
+
+Tickets
+-------
+
+Tickets will go on sale in {{ date.tickets_live }}.
+
+As always, we keep ticket prices low so that the event is accessible to the widest range of people possible.
+However, if you can't afford the tickets and still wish to attend, we will launch our grants program soon after opening ticket sales.
```

Comment:
```suggestion
As always, we try to keep ticket prices low so that the event is accessible to the widest range of people.
If you can't afford the tickets and still wish to attend, we will launch our grants program soon after opening ticket sales.
```